cpp/integ/projector_test.cc

```diff
@@ -15,11 +15,15 @@
 #include <gtest/gtest.h>
 #include "arrow/memory_pool.h"
 #include "integ/test_util.h"
+#include "gandiva/gandiva_configuration.h"
 #include "gandiva/projector.h"
 #include "gandiva/tree_expr_builder.h"
 
 namespace gandiva {
 
+extern const char kByteCodeFilePath[];
```

Comment:
instead of exposing this everywhere, can we just modify llvm_generator to use this path if the configuration isn't provided, or put this in the default configuration ?